app/views/work_orders/_form.html.erb

```diff
@@ -1,166 +1,46 @@
-<div class="col-xs-12">
-    <div class="row content-box">
-        <%= form_for [@job, @work_order] do |f| %>
-        <% if @work_order.errors.any? %>
-        <div id="error_explanation">
-            <h2><%= pluralize(@work_order.errors.count, "error") %>
-                prohibited this work_order from being saved:</h2>
-
-            <ul>
-                <% @work_order.errors.full_messages.each do |message| %>
-                <li><%= message %></li>
-                <% end %>
-            </ul>
-        </div>
+<%= simple_form_for [@job, @work_order] do |f| %>
+  <% if @work_order.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@work_order.errors.count, "error") %>
+          prohibited this work_order from being saved:</h2>
+      <ul>
+        <% @work_order.errors.full_messages.each do |message| %>
+          <li><%= message %></li>
         <% end %>
-        <hr/>
-        <div class="col-xs-12">
-          <h2>SERVPRO of <%= @job.try(:franchise).try(:name) %></h2>
-          <h2><%= @job.try(:franchise).try(:phone) %></h2>
-        </div>
-
-        <div class="col-sm-6">
-
-            <div class="form-group">
-                <%= f.label :to %><br>
-                <%= f.text_field :to, class: "form-control" %>
-            </div>
-        </div>
-        <div class="col-sm-6">
-            <div class="form-group">
-                <%= f.label :name %><br>
-                <%= f.text_field :name, class: "form-control" %>
-            </div>
-        </div>
-        <div class="col-sm-6">
-            <div class="form-group">
-                <%= f.label :date %><br>
-                <%= f.date_select :date %>
-            </div>
-        </div>
-        <div class="col-sm-6">
-            <div class="form-group">
-                <%= f.label :typed_by %><br>
-                <%= f.text_field :typed_by, class: "form-control", value: current_user.full_name %>
-            </div>
-        </div>
-        <div class="col-xs-12">
-          <hr/>
-        </div>
+      </ul>
+    </div>
+    <hr/>
+  <% end %>
+  <div class="col-xs-12">
+    <h2>SERVPRO of <%= @job.try(:franchise).try(:name) %></h2>
```

Comment:
Great suggestion thanks! I didn't want to take refactoring too far initially, and hate all the chained trys too (they were just moved from the initial code in this case), but moving these to delegates would be cleaner for sure.

Thanks!